lisk-dex-electron/src/Utils.js

```diff
@@ -1,3 +1,35 @@
+// Eventually store all config in singleton class and make it accessible everywhere
+const CryptoAsset = (() => {
```

Comment:
Good idea to have the config as an instance with some convenience methods but it would be better to have a single config object for all configs (not a separate one for assets); it's easier to manage and make sense of the code.

The method `getUnitValue(assetName)` could be renamed to `getAssetUnitValue(assetName)`.

This object could be renamed to just `Config` or `Configuration` and it can be returned by the `processConfiguration` function. For the tests, it can be constructed manually without the `processConfiguration` function.

Reply:
Will be moving ```Configuration``` to ```processConfiguration``` as a part of next PR